routes/valueRouter.js

```diff
@@ -29,13 +51,64 @@ valueRouter.get('/:itemId', async (req, res) => {
 // CREATE a new item
 valueRouter.post('/', async (req, res) => {
   try {
-    const { itemName, quantityType, quantity, price } = req.body;
-    const newItem = await db.query(
-      'INSERT INTO fair_market_value (item_name, quantity_type, quantity, price) VALUES ($(itemName), $(quantityType), $(quantity), $(price)) RETURNING *',
-      { itemName, quantityType, quantity, price },
+    const { itemName, quantityType, price, category } = req.body;
+    await db.query(
+      'INSERT INTO fair_market_value (item_name, quantity_type, price, category) VALUES ($(itemName), $(quantityType), $(price), $(category)) RETURNING *',
+      { itemName, quantityType, price, category },
+    );
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+// edit item
+valueRouter.put('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const { itemName, quantityType, price, category } = req.body;
+
+    await db.query(
+      `UPDATE fair_market_value
+      SET item_name = $(itemName), quantity_type = $(quantityType), price = $(price), category = $(category)
+      WHERE item_id = $(id) RETURNING *`,
+      { id, itemName, quantityType, price, category },
+    );
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
```

Comment:
check if values are present before inserting into table, so that if someone deletes the value in the edit modal, it won't get deleted from the table